### DISCUSSION TITLE
Prisma Freebsd 14 not found

### DISCUSSION THREAD

**Discussion by ShadowGaming100** [prisma/prisma#25936] [Q&A] [answered]
### Question

Does it support freebsd 14 , I check few pull request and issues I see there is support for freebsd but it can’t download it I tried to ignore checksum still the same thing

### Information about Prisma Schema, Client Queries and Environment (optional)

Precompiled engine files are not available for freebsd14. Read more about building your own engines at https://pris.ly/d/build-engines
> Downloading Prisma engines for Node-API for freebsd14 [] 0%Error: Failed to fetch sha256 checksum at https://binaries.prisma.sh/all_commits/605197351a3c8bdd595af2d2a9bc3025bca48ea2/freebsd14/schema-engine.sha256 - 404 Not Found

If you need to ignore this error (e.g. in an offline environment), set the PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING environment variable to a truthy value.
Example: PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1
ERROR: "build:schema" exited with 1.

**Comment by ShadowGaming100** [comment]
Looks like is no longer supported how do I build my self and specify the path

**Reply by nurul3101** [reply, marked as answer]
@ShadowGaming100 Here are the instructions for building Prisma Engines: https://github.com/prisma/prisma-engines?tab=readme-ov-file#building-prisma-engines

You can specify the path with these env variables: [PRISMA_QUERY_ENGINE_BINARY](https://www.prisma.io/docs/orm/reference/environment-variables-reference#prisma_query_engine_binary), [PRISMA_QUERY_ENGINE_LIBRARY](https://www.prisma.io/docs/orm/reference/environment-variables-reference#prisma_query_engine_library)

**Reply by ShadowGaming100** [reply]
Thanks for helping
